diff --git a/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/CheckboxWrapper.jsx b/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/CheckboxWrapper.jsx
index 806c698..617cb60 100644
--- a/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/CheckboxWrapper.jsx
+++ b/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/CheckboxWrapper.jsx
@@ -22,7 +22,7 @@ const CheckboxWrapper = styled(Box)`
   }
 
   /* Show active style both on hover and when the action is selected */
-  ${(props) => props.isActive && activeCheckboxWrapperStyles}
+  ${(props) => props.$isActive && activeCheckboxWrapperStyles}
   &:hover {
     ${activeCheckboxWrapperStyles}
   }
diff --git a/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/SubCategory.jsx b/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/SubCategory.jsx
index 76f2a7f..451dd13 100644
--- a/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/SubCategory.jsx
+++ b/node_modules/@strapi/plugin-users-permissions/admin/src/components/Permissions/PermissionRow/SubCategory.jsx
@@ -79,7 +79,7 @@ const SubCategory = ({ subCategory }) => {
 
             return (
               <Grid.Item col={6} key={action.name} direction="column" alignItems="stretch">
-                <CheckboxWrapper isActive={isActionSelected(action.name)} padding={2} hasRadius>
+                <CheckboxWrapper $isActive={isActionSelected(action.name)} padding={2} hasRadius>
                   <Checkbox
                     checked={get(modifiedData, name, false)}
                     name={name}
