diff --git a/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.js b/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.js
index 0f800da..bfc2591 100644
--- a/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.js
+++ b/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.js
@@ -5,6 +5,12 @@ function getOpenValues(options, defaultValue = {}) {
     const { value } = defaultValue;
     const option = options.find((option)=>option.value === value);
     if (!option) {
+        const rootOption = options.find((option)=>option.value === null);
+        if (rootOption?.children?.length) {
+            return [
+                rootOption.value
+            ];
+        }
         return values;
     }
     values.push(option.value);
diff --git a/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.mjs b/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.mjs
index db588c1..0f745e4 100644
--- a/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.mjs
+++ b/node_modules/@strapi/upload/dist/admin/components/SelectTree/utils/getOpenValues.mjs
@@ -3,6 +3,12 @@ function getOpenValues(options, defaultValue = {}) {
     const { value } = defaultValue;
     const option = options.find((option)=>option.value === value);
     if (!option) {
+        const rootOption = options.find((option)=>option.value === null);
+        if (rootOption?.children?.length) {
+            return [
+                rootOption.value
+            ];
+        }
         return values;
     }
     values.push(option.value);
